roles/dns_resolv/README.md

```diff
@@ -0,0 +1,53 @@
+dns_resolv
+===========
+
+Sets the default DNS resolve servers for the system
```

Comment:
With `blockinfile`, I don't think this describes what the role actually does.

```diff
$ diff -u resolv.conf.before resolv.conf.after 
--- resolv.conf.before  2020-07-08 15:38:35.968386005 -0400
+++ resolv.conf.after   2020-07-08 15:38:26.671639705 -0400
@@ -1,2 +1,7 @@
 nameserver 1.2.3.4
 search example.com
+# BEGIN ANSIBLE MANAGED BLOCK
+nameserver 208.67.222.222
+nameserver 1.1.1.1
+nameserver 8.8.8.8
+# END ANSIBLE MANAGED BLOCK
```

This should probably say something more along the lines of "Adds additional nameservers to a system's resolv.conf".

It may also be helpful (or it may just be extra noise) to mention that if the system's resolver config is managed by some service, like NetworkManager or resolvconf, changes made in an ansible play using this role may get reverted.